SocketHttpListener/WebSocket.cs

```diff
@@ -423,7 +423,7 @@ private bool processFragments(WebSocketFrame first)
 
         private bool processPingFrame(WebSocketFrame frame)
         {
-            var mask = Mask.Unmask;
+            //var mask = Mask.Unmask;
```

Comment:
Maybe just delete it? Why keep garbage...

Reply:
IDK, why was it there in the first place?

Reply:
Code sometimes rots... :(